packages/admin/config/filament.php

```diff
@@ -211,6 +211,25 @@
         ],
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Shortcuts
+    |--------------------------------------------------------------------------
+    |
+    | Using Mousetrap, you can bind keyboard shortcuts to your admin panel.
+    | Ctrl|Cmd + S will save the create or edit form
+    | Ctrl|Cmd + 0 will open the create form from the list page
```

Comment:
@buzkall shouldn't this `0` be `o`?
```suggestion
    | Ctrl|Cmd + O will open the create form from the list page
```

Reply:
My bad. Fixed now